### PR TITLE
Add task_01_03.py and test_01_03.py.

### PR DESCRIPTION
**Pull request by mukhaw**

### CODE REVIEW

File: hm_1/tasks/task_01_03.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        ...
+"""
+import os
+from typing import Tuple
+
+
+def find_maximum_and_minimum(file_name: str) -> Tuple[int, int]:
+    try:
+        if os.path.getsize(file_name) > 0:
+            file = open(file_name, "r")
```
Comment (Cjkjvfnby) [inline hm_1/tasks/task_01_03.py:18]: What is a recommended way to work with the file?

File: test/sample1.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+-1,2,3,4,-9,10,3,5,6,113
```
Comment (Cjkjvfnby) [inline test/sample1.txt:1]: Please read the task description carefully.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: hm_1/tasks/task_01_03.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+"""
+from typing import Tuple
+
+
+def find_maximum_and_minimum(file_name: str) -> Tuple[int, int]:
+    data = []
+    with open(file_name) as fi:
+        for line in fi:
+            data.append(int(line.rstrip("\n")))
+        data.sort()
```
Comment (Cjkjvfnby) [inline hm_1/tasks/task_01_03.py:19]: What is the complexity of this line?  Can we do better?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: hm_1/tasks/task_01_03.py (hunk 3 above)
Comment (mukhaw) [reply hm_1/tasks/task_01_03.py:19]: I don't know how to do this O(1)

Comment (Cjkjvfnby) [reply hm_1/tasks/task_01_03.py:19]: > I think It's O(len(fi))

It is not an exam, where I check your knowledge, it is a learning process.  I expect that you check your answer in the documentation.

> I don't know how to do this O(1)

It is not possible, O(l) where l is the number of lines in files expected here.

File: hm_1/tasks/task_01_03.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        return "This file is empty" if not data else (data[-1], data[0])
```
Comment (Cjkjvfnby) [inline hm_1/tasks/task_01_03.py:20]: Please pay attention to the task definition. Can the file be empty?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**